pydata_sphinx_theme/__init__.py

```diff
@@ -196,4 +196,9 @@ def setup(app):
     app.connect("html-page-context", setup_edit_url)
     app.connect("html-page-context", add_toctree_functions)
 
+    # Update templates for sidebar
+    pkgdir = os.path.abspath(os.path.dirname(__file__))
+    path_templates = os.path.join(pkgdir, "_templates")
+    app.config.templates_path.append(path_templates)
```

Comment:
just wondering about this one: is there some documentation in sphinx that this is the recommended way to add custom templates as a package?

Reply:
hmmm, well it's documented in a way that suggests you could add to it to increase the search scope for templates: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-templates_path

and it seems this is how Sphinx adds templates in its own codebase, e.g.: https://github.com/sphinx-doc/sphinx/blob/cc25bac26445688c50efb9b8d5f30571940e27bd/sphinx/ext/autosummary/generate.py#L644

Reply:
If that's how they do it for the built-in extensions, that's certainly fine!